cli/metrics/conn_windows.go

```diff
@@ -30,6 +31,10 @@ var (
 	socket = `\\.\pipe\docker_cli`
 )
 
+func init() {
+	overrideSocket() // nop, unless built for e2e testing
```

Comment:
Did you mean
```suggestion
	overrideSocket() // no-op, unless built for e2e testing
```
or something like that?

Reply:
`nop` is the mnemonic for no-op in most assemblers (and tends to be used as a neologism for "do nothing"), but yeah that would be clearer.

Reply:
done :)